src/SDKs/Search/DataPlane/Search.Tests/Tests/EncryptionWithCMK.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Search.Tests
+{
+    using Microsoft.Azure.Search.Models;
+    using Microsoft.Azure.Search.Tests.Utilities;
+    using Xunit;
+
+    public sealed class EncryptionWithCMKTests : SearchTestBase<EncryptionFixture>
+    {
+        [Fact]
+        public void CreateIndexWithCMKUsingAccessCredentials()
+        {
+            Run(() =>
+            {
+                SearchServiceClient searchClient = Data.GetSearchServiceClient();
+
+                Index encryptedIndex = CreateEncryptedTestIndex(new EncryptionKey()
```

Comment:
I wanted to call this out as an example of test code done right. Your instinct in production code might be to factor these details out into a method, but in a test like this having the details be explicit clearly communicates the intended usage of the API. This test communicates exactly what it needs to.